plugins/scaffolder-backend-module-gitea/src/actions/gitea.ts

```diff
@@ -112,12 +113,23 @@ const createGiteaProject = async (
     This is the default scenario that we support currently
   */
   let response: Response;
+  let visibility: boolean;
```

Comment:
```suggestion
  let private: boolean;
```

Reply:
Think this is a bit clearer for the API call right?

Reply:
According to typescript "private" private is a reserved word in strict mode. So, can we use it or not ?

`TS1214: Identifier expected. private is a reserved word in strict mode. Modules are automatically in strict mode.`

Reply:
Maybe `isPrivate` works better then?

Reply:
I vote for `isPrivate` :-)

Reply:
Let's go with that then! 😄

Reply:
Do you plan to commit the change using `isPrivate` as suggestion or should I do it ?

Reply:
@cmoulliard I can't push to this branch, so you'd have to do it :)